### PR TITLE
products: -added products-page to e-com  (non-functional)

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
-added products-page to e-com  (non-functional)
-added products navbar

image

### CODE REVIEW

File: src/components/products/productsCard.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.product-card-container {
+    margin: 1rem 0.5rem;
+    background-color: var(--white);
+    border-radius: 5px;
+    height: auto;
+}
+
+.product-card_container-images {
```
Comment (Raghav888) [inline src/components/products/productsCard.css:14]: Good naming convention

File: src/components/products/productsCard.jsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            </p>
+            <div className="card_btn">
+              <button className="btn-primary">Buy Now</button>
+              <button className="btn-outline">Wishlist</button>
+            </div>
+          </div>
+          <div className="product-card-container">
+            <img
+              src="https://i.ibb.co/ZY0WRzs/Indian-traditional-jewerly-and-clothing-accessories.jpg"
+              className="product-card_container-images"
```
Comment (Raghav888) [inline src/components/products/productsCard.jsx]: You can use object and iterate over it that will reduce your code length

File: src/components/products/productsCard.jsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+              alt=""
```
Comment (Raghav888) [inline src/components/products/productsCard.jsx]: put value in alt tag, that's good practice

File: src/components/products/productsFilter.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+                <h2 className="headings">Category</h2>
+                <p> <input type="checkbox" name="category"/>&nbsp;Gadgets</p>
+                <p> <input type="checkbox" name="category"/>&nbsp;Men-Collection</p>
+                <p> <input type="checkbox" name="category"/>&nbsp;Kids-Collection</p>
+                <p> <input type="checkbox" name="category"/>&nbsp;Women-Collection</p>
+            </div>
+            <div className="ratings-filter filter">
+                <h2 className="headings">Ratings</h2>
+                <p> <input type="radio" name="rating"/>&nbsp;4&nbsp;<i className="fa-solid fa-star"></i>&&nbsp;Above</p>
+                <p> <input type="radio" name="rating"/>&nbsp;3&nbsp;<i className="fa-solid fa-star"></i>&&nbsp;Above</p>
```
Comment (Raghav888) [inline src/components/products/productsFilter.jsx]: Same here you can use array here

**Review by Raghav888 — APPROVED** [review]

**Review by neetukumari4858 — APPROVED** [review]
overall looks good and I like that you have broken large component into small components.

**Pull request merged**